sbi/utils/user_input_checks_utils.py

```diff
@@ -25,17 +24,28 @@ def __init__(
         return_type: Optional[torch.dtype] = float32,
         batch_shape=torch.Size(),
         event_shape=torch.Size(),
-        validate_args=None,
+        validate_args=False,
+        lower_bound: Optional[Tensor] = None,
+        upper_bound: Optional[Tensor] = None,
     ):
         super().__init__(
             batch_shape=batch_shape,
             event_shape=event_shape,
             validate_args=validate_args,
         )
 
+        if not validate_args:
+            warnings.warn(
```

Comment:
So, if the user does not explicitly pass `validate_args`, they will get this warning, right? And if they pass `lower_bound`, but not `validate_args`, they will also get the warning?